bazel/emscripten_toolchain/link_wrapper.py

```diff
@@ -54,8 +43,41 @@
 if rtn != 0:
   sys.exit(1)
 
-js_name = os.path.basename(js_output)
-base_name = os.path.splitext(js_name)[0]
+# Parse the arguments that we gave to the linker to determine what the output
+# file is named and what the output format is.
+parser = argparse.ArgumentParser(add_help=False)
+parser.add_argument('-o')
+parser.add_argument('--oformat')
+(options, _) = parser.parse_known_args(param_file_args)
```

Comment:
Maybe `options = parser.parse_known_args(param_file_args)[0]`  (or at least drop the braces on the left hand side of the expression.